client-bagel-buddy/src/components/popup/index.css

```diff
@@ -0,0 +1,66 @@
+
+.popup-overlay {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    z-index: 1000;
+}
+
+.popup-container {
+    border-radius: 24px;
+    background: var(--bg-white, #FFF);
+    display: flex;
+    width: 600px;
+    padding: 24px;
+    gap: 24px;
+}
+
+.popup-container .close-button {
+    cursor: pointer;
+    border: none;
+    background-color: white;
+    font-size: 18px;
+    float: right;
+}
+
+.popup-button {
+    font-weight: 700;
+    font-size: 20px;
+    padding: 12px 24px;
+    border-radius: 12px;
+    margin: 12px 0px;
+    color: white;
```

Comment:
.popup-button is the same as `btn main`. Hence, no need to define the class again. Can reomve it here and change the className in js to `btn main`